packages/styles/components/_c.dropdown.scss

```diff
@@ -52,19 +52,14 @@
 
     // invalid state
     .is-invalid + & {
-      right: ($mu200 + $mu025) * 2; // =72px
+      right: px-to-rem(72);
     }
   }
 
   &__trigger {
```

Comment:
Can be merged with line 10

Reply:
Thanks for your feedback @mohamedMok .

It's fixed ;)